src/app/Dashboard/DraggableRef.tsx

```diff
@@ -35,125 +35,327 @@
-import { DashboardCardSizes } from './Dashboard';
-import { DashboardCardContext } from './ResizableCard';
+import { useDispatch } from 'react-redux';
+
+type itemPosition = 'left' | 'right' | 'else';
+//
+// |-----------------| mouse |-----------------|
+// |   rect1.right   ^       ^   rect2.left    |
+// |                 |       |                 |
+// returns [inBetweenTwoRectangles, {insertedOnLeft, insertedOnRight, else}]
+function inBetween(ev: MouseEvent, rect1: DOMRect, rect2: DOMRect): [boolean, itemPosition] {
+  const withinHeightRect1 = ev.clientY > rect1.top && ev.clientY < rect1.bottom;
+  const withinHeightRect2 = ev.clientY > rect2.top && ev.clientY < rect2.bottom;
+  // Cases
+  // same row -> between: no ends
+  const singleRowBetween =
+    rect1.top === rect2.top &&
+    rect1.right <= ev.clientX + rect1.width * offsetScale &&
+    ev.clientX - rect2.width * offsetScale <= rect2.left &&
+    withinHeightRect1;
+  // same row -> before: left end
+  const singleRowBefore =
+    rect1.top === rect2.top && ev.clientX <= rect2.left && rect1.left >= rect2.right && withinHeightRect1;
+  // same row -> after: right end
+  const singleRowAfter =
+    rect1.top === rect2.top && ev.clientX >= rect1.right && rect1.right >= rect2.left && withinHeightRect1;
+
+  // different rows -> before: left end
+  const multRowBefore = rect1.top !== rect2.top && ev.clientX <= rect2.left && withinHeightRect2;
+  if (multRowBefore || singleRowBefore) {
+    return [true, 'left'];
+  }
+  // different rows -> after: right end
+  const multRowAfter = rect1.top !== rect2.top && ev.clientX >= rect1.right && withinHeightRect1;
+  if (multRowAfter || singleRowAfter) {
+    return [true, 'right'];
+  }
+  // different rows -> between: no ends
+  return [singleRowBetween, 'else'];
+}
+
+function getOrder(el: HTMLElement): number {
+  return parseInt(el.getAttribute(dashboardCardOrderAttribute) as string);
+}
+
+const initStyle = {};
+
+const transition = 'transform 0.5s cubic-bezier(0.2, 1, 0.1, 1) 0s';
+const delayedTransition = 'transform 0.5s cubic-bezier(0.2, 1, 0.1, 1) 0.3s';
+const overlapTranslateY = -15;
+const translateX = 50;
+const offsetScale = 0.33;
+
+function overlaps(ev: MouseEvent, rect: DOMRect): boolean {
+  return (
+    ev.clientX - rect.width * offsetScale > rect.left &&
+    ev.clientX + rect.width * offsetScale < rect.right &&
+    ev.clientY > rect.top &&
+    ev.clientY < rect.bottom
+  );
+}
+
+export const dashboardCardOrderAttribute = 'dashboard-card-order';
+
+interface DroppableItem {
+  index: number;
+  node: HTMLElement;
+  rect: DOMRect;
+  isDraggingHost: boolean;
+  isHovered: boolean;
+}
+
+function resetDroppableItem(droppableItem: DroppableItem) {
+  droppableItem.node.style.transition = transition;
+  droppableItem.node.style.transform = '';
+  droppableItem.isHovered = false;
+}
+
+function setDroppableItem(droppableItem: DroppableItem, transition: string, transform: string, isHovered?: boolean) {
```

Comment:
Try setting its `opacity: 0.5`. This will allow cards "under" to fade a bit on hovered.

Reply:
![Screenshot from 2023-01-26 14-05-14](https://user-images.githubusercontent.com/68053619/214926652-011255c2-1fa1-4e41-ab18-3844806dc172.png)